sql/hive/src/main/scala/org/apache/spark/sql/hive/orc/OrcOptions.scala

```diff
@@ -42,7 +44,7 @@ private[orc] class OrcOptions(
   val compressionCodec: String = {
     // `compression`, `orc.compress`, and `spark.sql.orc.compression.codec` are
     // in order of precedence from highest to lowest.
```

Comment:
Please update the comment.

Reply:
Thank you for review, @gatorsmile .
The name `orc.compress` and precedence order is not changed. Which part do you want to change?

Reply:
`COMPRESS.getAttribute` is not in our code base. Please change it to
> // `compression`, `orc.compress`(i.e., OrcConf.COMPRESS), and `spark.sql.orc.compression.codec` are

Reply:
I see.